packages/docs/docs/medications/representing-prescriptions-and-medication-orders.md

```diff
@@ -0,0 +1,96 @@
+import MedplumCodeBlock from '@site/src/components/MedplumCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import ExampleCode from '!!raw-loader!@site/..//examples/src/medications/representing-prescriptions-and-medication-orders.ts';
+
+# Representing Prescriptions and Medication Orders
+
+Prescriptions and medication orders are very common in healthcare settings and accuracy is vitally important. This guide will go over how to represent all aspects of a medication order, from the request to the administration instructions. Note that this is an _operational counterpart_ to the [Modeling A Formulary](https://www.medplum.com/docs/medications/formulary) documentation.
```

Comment:
We should use relative links e.g. [Modeling A Formulary](/docs/medications/formulary)

Reply:
The links have been updated